### ISSUE TITLE
Export Orgs from EasyCLA with addresses

### ISSUE THREAD

**Issue by mlehotskylf** [communitybridge/easycla#4475]
Export Orgs from EasyCLA with addresses

It should have these column:

- Org name
- Org adress
- Org ID
- number of CCLA signed (int)
- date of first CCLA signed (date)
- date of last CCLA signed (date)
- list of CLA_GROUP_NAMEs (list)
- list of CLA_MANAGERS (list)

**Comment by mlehotskylf** [comment]
Please add column showing account ID (SFDC) for orgs if we have it? I believe we are calling it `company_external_id` in EasyCLA DB.

**Comment by nickmango** [comment]
[2024-10-15 12_26pm (2).csv](https://github.com/user-attachments/files/17645834/2024-10-15.12_26pm.2.csv)

@mlehotskylf I have attached the orgs with the above details aside from the address thats failing to get parsed. Iam currently blocked on that

**Comment by nickmango** [comment]
[company_data_2024-11-11T23:51:50.csv](https://github.com/user-attachments/files/17708139/company_data_2024-11-11T23.51.50.csv)
@mlehotskylf attached is the updated csv of the companies with the addresses, Some have missing docusign details

**Comment by nickmango** [comment]
[company_data_2024-11-12T15:09:21.csv](https://github.com/user-attachments/files/17716344/company_data_2024-11-12T15.09.21.csv)
Updated list with companies that have signed ccla records

**Comment by mlehotskylf** [comment]
@nickmango As we discussed, company address is not populating when org has multiple CCLAs, some with address and some without. However, lets put this on hold for now as we may be able to get addresses for these orgs from Salesforce.

**Comment by mlehotskylf** [comment]
I believe this is no longer needed. Closing.
